vue/components/ui/molecules/slider-ripe/slider-ripe.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <input-ripe
+        class="slider-ripe"
+        v-bind="mergedInputProps"
+        v-bind:value.sync="valueData"
+        v-bind:type="'range'"
+        v-on:update:value="onInput"
+        v-on:blur="onBlur"
+        v-on:focus="onFocus"
+    />
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.slider-ripe {
+    appearance: none;
+    border: none;
+    height: 20px;
+    margin: 0px 0px 0px 0px;
+    padding: 0px 0px 0px 0px;
+}
+
+.slider-ripe::-webkit-slider-runnable-track {
+    background: #57626e;
+    border-radius: 4px;
+    cursor: pointer;
+    height: 3px;
+}
+
+.slider-ripe::-moz-range-track {
+    background: #57626e;
+    border-radius: 4px;
+    cursor: pointer;
+    height: 3px;
+}
+
+.slider-ripe::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    background: #57626e;
+    border: 2px solid $light-white;
+    border-radius: 50%;
+    cursor: col-resize;
+    height: 16px;
+    margin-top: -7px;
+    width: 16px;
+}
+
+.slider-ripe:focus::-webkit-slider-thumb {
+    border-color: $aqcua-blue;
+}
+
+.slider-ripe::-moz-range-thumb {
+    background: #57626e;
+    border: 2px solid $light-white;
+    border-radius: 50%;
+    cursor: col-resize;
+    height: 12px;
+    width: 12px;
+}
+
+.slider-ripe:focus::-moz-range-thumb {
+    border-color: $aqcua-blue;
+}
+</style>
+
+<script>
+export const SliderRipe = {

```

Comment:
Can't this just be called `Slider`?

Reply:
Yes, I named it that way because in the main issue it was suggested slider-ripe. Renaming